sceneview/src/main/java/io/github/sceneview/node/CubeNode.kt

```diff
@@ -25,7 +21,7 @@ open class CubeNode(
      * Should return the material to bind for the zero-based index of the primitive, must be less
      * than the [Geometry.submeshes] size passed to constructor.
      */
-    materialInstances: (index: Int) -> MaterialInstance? = { materialInstance },
+    materialInstances: (index: Int) -> MaterialInstance,
```

Comment:
Why moving it to non nullable?
It could be usefull in case only some faces requiring material.
Same for every Geometries they could be used for collision purposes.

Reply:
I thought a face must be attached with a material. It is fixed now.